_episodes/33.md

```diff
@@ -0,0 +1,266 @@
+different sources available in one spot for our data science team was a tall 
+order. Data must be quickly accessible to data consumers and systems like Trino
+must be reliable for users to trust this singular access point for their data.
+
+In order for analysts and data scientists to use these services, they first need
+to trust in the system. It was vital to Goldman Sachs that Trino have High
+Availability so that in the event of failure, there was always another Trino
+cluster available to take requests.
+
+### Integrating Trino into the Goldman Sachs internal ecosystem 
+
+Before high availability was a concern, the team had to first integrate Trino to
+meet their requirements. This included integrating with internal security 
+systems, observability systems, and credential stores. It also meant
+adding integration with their governance services that manage cataloguing
+services and data discovery engines. Finally, while many of the connectors that
+the team intended to use existed in the open source. There were many missing 
+features and performance enhancements that would lead to a better user 
+experience and more adoption. The team has since taken it upon themselves to
+work on these features and contribute them back to Trino. We will cover some of 
+these contributions in the PR segment of this show.
+
+### Achieving scaling and high availability
+
+Once the team had much of Trino running for some initial use cases, the next 
+step was to improve support for more simultaneous use cases and highly 
+concurrent workloads. The team wanted trust in the system and so as they scaled
+the ability to run blue-green deployments, enable resources isolation, and
+have highly available clusters through failures became much more pertinant.
+
+### Trino ecosystem at Goldman Sachs
+
+Here is an overview of the Goldman Sachs Ecosystem. It showcases the preexisting
+services that needed to connect to Trino, the catalogs supported, and the method
+in which Goldman Sachs acheives High Availability through supporting multiple
+clusters in various groups.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/33/trinoecosystem.png"/><br/>
+Source: <a href="https://developer.gs.com/blog/posts/enabling-highly-available-trino-clusters-at-goldman-sachs">Goldman Sachs Blog</a>
+</p>
+
+### Dynamic query routing
+
+In order to ensure that all the clusters receive an even distribution the team
+created services that enable dynamic query routing across the different cluster
+groups.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/33/trinodynamicqueryrouting.png"/><br/>
+Source: <a href="https://developer.gs.com/blog/posts/enabling-highly-available-trino-clusters-at-goldman-sachs">Goldman Sachs Blog</a>
+</p>
+
+### Query routing components
+
+* [Envoy](https://www.envoyproxy.io/) Proxy - open source edge and service proxy
```

Comment:
use

```
[Envoy Proxy](https://www.envoyproxy.io/)
```

or drop "Proxy"

Reply:
also .. do we want to mention that this is a CNCF project?

Reply:
meh, they can find that out when they go to the site no?